src/components/apps/launch/ReferenceGenomeSelect.js

```diff
@@ -7,17 +7,29 @@
  */
 import React from "react";
 import { useTranslation } from "i18n";
+
+import { useReferenceGenomes } from "../queries";
+
 import { FormFieldLoading, FormTextField } from "@cyverse-de/ui-lib";
 
 import { MenuItem } from "@material-ui/core";
 
-const ReferenceGenomeSelect = ({
-    referenceGenomes,
-    referenceGenomesLoading,
-    ...props
-}) => {
+const ReferenceGenomeSelect = ({ param, ...props }) => {
+    const [referenceGenomes, setReferenceGenomes] = React.useState([]);
+
+    const { isFetching: referenceGenomesLoading } = useReferenceGenomes(
+        true,
+        setReferenceGenomes
+    );
```

Comment:
An odd side effect of moving the listing query into the reference genome selector components, is that when a form with multiple selectors first loads, each selector appears to be loading the listing (as intended), but if the view changes, and the selectors are hidden and then re-shown, then only the first selector in the form appears to be loading again, but the remaining selectors apparently are getting a `false` value for `isFetching`, but also an empty listing, so they will display an error message that the listing couldn't be loaded.

It's only temporary though. Once the first selector loads, they all refresh and load the listing as normal.

You can test this in Storybook with the "kitchen sink" test app: https://5ebad5c6b991d60022b1e72e-cgpehakrzf.chromatic.com/?path=/story/apps-editor--kitchen-sink-editor
Go to step 2 and quickly scroll down to see the 3 selectors loading, then click the "Next" button at the bottom to go to the preview step. The view should stay scrolled to the bottom of the preview where you can see the first selector reloading, and the last 2 showing an error until the first one finishes loading; then they all appear as normal.

I'm not sure if I should be checking something else besides `isFetching` here (the `isLoading` flag appears to only be set when the list is initially loaded from the API, but not after its cached).